jumeg/connectivity/con_viz.py

```diff
@@ -734,25 +750,19 @@ def get_vmin_vmax_causality(vmin, vmax, cau_l, cau_u):
         The maximum value.
     """
 
+    # off diagonal elements
+    cau_od = cau[np.where(~np.eye(cau.shape[0], dtype=bool))]
     if vmax is None:
-        vmax = np.max([np.max(cau_l), np.max(cau_u)])
+        vmax = cau_od.max()
+
     if vmin is None:
-        if np.max([np.max(cau_l), np.max(cau_u)]) == 0:
+        if cau_od.max() == 0:
             # no significant connections found
             vmin = 0
             vmax = 0.2
         else:
-
-            if (cau_l != 0).any():
-                vmin_l = np.min(cau_l[cau_l != 0])
-            else:
-                vmin_l = 0
-
-            if (cau_u != 0).any():
-                vmin_u = np.min(cau_u[cau_u != 0])
-            else:
-                vmin_u = 0
-            vmin = np.min([vmin_l, vmin_u])
+            # get minimum value that is different from 0
+            vmin = cau_od[np.where(cau_od)].min()
```

Comment:
This fails when there are no connections (i.e. all zeros). It should still plot the matrix, but plot no connections. 

```
Traceback (most recent call last):
  File "plot_grouped_connectivity_circle.py", line 51, in <module>
    n_lines=5, colorbar=True)
  File "/Users/psripad/miniconda3/lib/python3.7/site-packages/jumeg/connectivity/con_viz.py", line 515, in plot_grouped_connectivity_circle
    title=title, **kwargs)
  File "/Users/psripad/miniconda3/lib/python3.7/site-packages/jumeg/connectivity/con_viz.py", line 332, in plot_connectivity_circle
    vmin = np.min(con[np.abs(con) >= con_thresh])
  File "<__array_function__ internals>", line 6, in amin
  File "/Users/psripad/miniconda3/lib/python3.7/site-packages/numpy/core/fromnumeric.py", line 2793, in amin
    keepdims=keepdims, initial=initial, where=where)
  File "/Users/psripad/miniconda3/lib/python3.7/site-packages/numpy/core/fromnumeric.py", line 90, in _wrapreduction
    return ufunc.reduce(obj, axis, dtype, out, **passkwargs)
ValueError: zero-size array to reduction operation minimum which has no identity
```